src/py42/services/users.py

```diff
@@ -58,6 +59,7 @@ def create_user(
         except Py42InternalServerError as err:
             if u"USER_DUPLICATE" in err.response.text:
                 raise Py42UserAlreadyExistsError(err)
+            raise
```

Comment:
This is the fix for what is listed in the CHANGELOG

Reply:
wouldn't the `Py42UsernameMustBeEmailError` also make sense here?

Reply:
Hm yes I would think so, let me see

Reply:
Just checked -- we get a slightly different and less-specific error message in the response.

```
Response content: [{"name":"USERNAME_INVALID","description":"An error has occurred. See server logs for more information.","objects":[]}]
```

So it is not clear of the exact reasons from the error message alone